### DISCUSSION TITLE
[M-Z] (15 Gennaio 2018) Esercizio 8

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#503] [Esami & Esoneri M-Z]
### Traccia

<img width="874" alt="image" src="https://github.com/user-attachments/assets/c4880aac-759f-4f8a-8734-93b7c7de857a">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20180115.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20180115.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/3d66bdcf-a46a-4354-b626-8b75340f41c9)
